zkg

```diff
@@ -571,22 +571,22 @@ def cmd_test(manager, args, config, configfile):
             print_error(
                 str.format('error: failed to run tests for "{}": {}', name, error_msg)
             )
-            continue
```

Comment:
Please leave that in, Otto — this skips to the next package when there's a problem prepping the package for testing (so that's different from its tests failing). The `error_message` should be helpful regarding the cause, and the instructions below for the `zkg.test_command` files don't apply in that case since nothing's been run yet. If you have a case where `error_message`  is missing or not helpful, then that's something to fix.

The fact that this change didn't trigger a test failure indicates that we're testing loosely here, which isn't good.

Reply:
No problem

Reply:
the error message is not helpful in the case I just saw, but this was:
```
[root@skid zeek-kafka]# pwd
/opt/zeek/var/lib/zkg/testing/zeek-kafka/clones/zeek-kafka
[root@skid zeek-kafka]# cat zkg.test_command.stderr
kafka.l2s-l2e-no-overlap ... failed
  % 'zeek ../../../scripts/Seiso/Kafka/ /opt/zeek/var/lib/zkg/testing/zeek-kafka/clones/zeek-kafka/tests/.tmp/kafka.l2s-l2e-no-overlap/l2s-l2e-no-overlap.zeek > output' failed unexpectedly (exit code 1)
  % cat .stderr
  error in /opt/zeek/share/zeek/base/init-bare.zeek, line 1: cannot load plugin library /opt/zeek/var/lib/zkg/testing/zeek-kafka/clones/zeek-kafka/build//lib/SEISO-KAFKA.linux-x86_64.so: /opt/zeek/var/lib/zkg/testing/zeek-kafka/clones/zeek-kafka/build//lib/SEISO-KAFKA.linux-x86_64.so: undefined symbol: _ZN4zeek6plugin6Plugin12HookLoadFileENS1_8LoadTypeERKNSt7__cxx1112basic_stringIcSt11char_traitsIcESaIcEEESA_
  fatal error in /opt/zeek/share/zeek/base/init-bare.zeek, line 1: aborting after plugin errors
```
is that in the testing_dir?

Reply:
error: failed to run tests for zeek/seisollc/zeek-kafka: test_command failed with exit code 1

is the error_message when this happens

Reply:
I don't think this change broke the tests because it doesn't change behavior, unless passed is true and error_message is not None

Reply:
but I changed it back and added a little more detail to the error printout, without going all the way to duplicating the clone dir stuff